app/assets/stylesheets/active_admin/_header.scss

```diff
@@ -103,7 +101,7 @@
       ul {
         background: $hover-menu-item-background;
         @include rounded-all(0,10px,10px,10px);
-        @include shadow(0, 1px, 3px, #444);
+        box-shadow: 0 1px 3px #444;
```

Comment:
Color literals like `#444` should only be used in variable declarations; they should be referred to via variable everywhere else.